src/omnisharp/OmnisharpManager.ts

```diff
@@ -20,10 +20,10 @@ export class OmnisharpManager {
         private platformInfo: PlatformInformation) {
     }
 
-    public async GetOmniSharpLaunchInfo(omnisharpPath: string, serverUrl: string, latestVersionFileServerPath: string, installPath: string, extensionPath: string): Promise<LaunchInfo> {
+    public async GetOmniSharpLaunchInfo(defaultOmnisharpVersion: string, omnisharpPath: string, serverUrl: string, latestVersionFileServerPath: string, installPath: string, extensionPath: string): Promise<LaunchInfo> {
```

Comment:
Consider pushing all of these parameters into some "omnisharpSettings" class to make their relationship clearer?

Reply:
Yes that will be a part of the next refactoring step that we will be taking up, where we will be introducing these settings classes.